src/components/cards/ValidationList/ValidationList.tsx

```diff
@@ -14,28 +14,47 @@ function ValidationList({
     <>
       {/* WIP: waiting on final design from lucas */}
       <div
-        className={!isMobile ? 'flex flex-column' : 'flex flex-column center'}
+        className={!isMobile ? 'flex flex-column' : 'flex flex-column left'}
         style={{ ...wrapperCustomStyle, zIndex: '1001' }}
       >
         {validations?.map((validationItem: ValidationObject, index: number) => (
           <span
             key={index}
-            className="flex flex-left text white"
+            className="flex flex-column flex-left text white"
             style={{
-              gap: '1em',
-              alignItems: 'center',
+              gap: '0px',
+              alignItems: 'flext-start',
+              justifyContent: 'flex-start',
             }}
           >
-            {validationItem.status ? (
-              <CircleCheck
-                width={20}
-                height={20}
-                fill={'var(--success-green)'}
-              />
-            ) : (
-              <CircleXIcon width={20} height={20} fill={'var(--error-red)'} />
-            )}
-            {validationItem.name}
+            <span
+              className="flex"
+              style={{
+                gap: '15px',
+                alignItems: 'center',
+              }}
+            >
+              {validationItem.status ? (
+                <CircleCheck
+                  width={20}
+                  height={20}
+                  fill={'var(--success-green)'}
+                />
+              ) : (
+                <CircleXIcon width={20} height={20} fill={'var(--error-red)'} />
+              )}
+              {validationItem.name}
+            </span>
+            <span
+              className="flex faded center"
+              style={{
+                gap: '15px',
+                alignItems: 'center',
+              }}
+            >
+              {validationItem.error?.toString() ??
+                validationItem.message?.toString()}
```

Comment:
where are to the `warning` tags i shared in my suggestions PR? didn't that get merged here?

Reply:
errors in validation list, warnings on input